languages/python/python.go

```diff
@@ -40,20 +49,64 @@ func (l language) GetLanguage() *sitter.Language {
 }
 
 func (l language) Imports(content []byte, root *sitter.Node) ([]string, error) {
+	for _, qi := range q {
+		if out := imports(content, root, qi); len(out) > 0 {
```

Comment:
This is incorrect. You should append all results together. What if you have an import that matches the first query and the second import that matches the second query?